provision/minikube/measure/collect-sut-info.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -e
+
+export timestamp="${timestamp:-$(date -uIseconds)}"
+
+sutInfoDir="data/${timestamp}/system-under-test"
+
+echo "Collecting information about system under test."
+
+echo "- pods/keycloak-0"
```

Comment:
could this be configurable to be fetching the information about other keycloak pods when the KC_SCALE value is greater than 1 ?